utilities/env_mirror.cc

```diff
@@ -27,7 +27,7 @@ class SequentialFileMirror : public SequentialFile {
     if (as == Status::OK()) {
       char* bscratch = new char[n];
       Slice bslice;
-      size_t off = 0;
+      size_t off __attribute__((__unused__)) = 0;
```

Comment:
Can this be removed?

Reply:
It is used in assert(). So the compiler only complain when `DEBUG_LEVEL=0`.

Reply:
Windows build is failing for that. How about we only define that with in `#ifndef NDEBUG`?

Reply:
Yeah, I think __attribute__ is gcc/clang thing